waspc/src/Wasp/Generator/JsImport.hs

```diff
@@ -1,26 +1,33 @@
 module Wasp.Generator.JsImport
-  ( getJsImportDetailsForExtFnImport,
+  ( extImportToJsImport,
+    RelDirToExternalCodeDir,
   )
 where
 
-import StrongPath (Dir, Path, Posix, Rel, (</>))
+import StrongPath (Dir, Dir', Path, Posix, Rel, (</>))
 import qualified StrongPath as SP
-import qualified Wasp.AppSpec.ExtImport as AS.ExtImport
+import qualified Wasp.AppSpec.ExtImport as EI
 import Wasp.Generator.ExternalCodeGenerator.Common (GeneratedExternalCodeDir)
+import Wasp.JsImport
+  ( JsImport,
+    JsImportName (JsImportField, JsImportModule),
+    makeJsImport,
+  )
+
+type RelDirToExternalCodeDir = Path Posix (Rel GeneratedExternalCodeDir) Dir'
```

Comment:
I'd like to keep the type since it's used in 4 places and helps not to drag around a more complex definition. 

The concept is "relative directory path from import location to external code dir" e.g. "../../" which says the import is happening two levels below external code dir. 

`RelDirFromImportLocationToExternalCodeDir` captures that for me.